pkg/trace/trace.go

```diff
@@ -66,6 +66,15 @@ func (t *Message) delta() time.Duration {
 	return time.Now().Sub(t.startTime)
 }
 
+// Add Syslog hook
+func InitLogger(cfg *log.LoggingConfig) error {
+	hook, err := log.GetSyslogHook(cfg)
+	if err == nil && hook != nil {
+		Logger.Hooks.Add(hook)
```

Comment:
I suspect we'll get away with this but if I recall correctly `Hooks.Add` is not thread safe.
Please confirm I'm correctly, that there's no risk of calling this method concurrently and note thread safety in the function comment.